src/Service/Context/PageContext.php

```diff
@@ -67,7 +74,6 @@ class PageContext {
     'account_name' => 'account_id',
```

Comment:
"account_name" and "customer_site" naming were inherited from lift2 codebase; they were also widely referred in [Lift Web](https://github.com/acquia/acquia-lift-web/blob/cd7394c4131cd2852720325b5320fbc0e68e896f/admin/src/main/java/com/trucentric/core/domain/dice/VariationAggregation.java#L59), [legacy docs](https://docs.acquia.com/lift/drupal/configure-lift) and [support infrastructures](https://cci.acquia.com/node/4452101/edit) key / meta data. acquia-lift-decision-api and lift3 have changed the naming to "account_id" and "site_id".

We have already asked about this naming unification before but there was no conclusion. I'll be happy to update to match "account_id" and "site_id", but I need to make sure 1) this is where we want to go and what we'd like to undertake, and 2) we have team consensus.